lambda-function-connect-rds-iam/example.js

```diff
@@ -0,0 +1,67 @@
+/* 
+Node.js code here.
+*/
+// ES6+ example
+import { Signer } from "@aws-sdk/rds-signer";
+import mysql from 'mysql2/promise';
+
+async function createAuthToken() {
+  // Define connection authentication parameters
+  const dbinfo = {
+
+    hostname: process.env.ProxyHostName,
+    port: process.env.Port,
+    username: process.env.DBUserName,
+    region: process.env.AWS_REGION,
+  
+  }
+
+  // Create RDS Signer object
+  const signer = new Signer(dbinfo);
+  
+  // Request authorization token from RDS, specifying the username
+  const token = await signer.getAuthToken(
+    {
+      username: process.env.DBUserName,
+    }
+  );
+  return token;
+}
+
+async function dbOps() {
+
+  try {
+      // Obtain auth token
+      const token = await createAuthToken();
+      // Define connection configuration
+      let connectionConfig = {
+        host: process.env.ProxyHostName,
+        user: process.env.DBUserName,
+        //password: token,
+        database: process.env.DBName,
+        ssl: 'Amazon RDS',
+        authPlugins: {
+          mysql_clear_password: () => () => Buffer.from(token)
```

Comment:
Why do we need this? Would be good to have a comment explaining why.

Reply:
Removed in favor of token